main.go

```diff
@@ -163,7 +207,7 @@ func discoverNewChannels(node CincinnatiNode, startChannelPrefix string, minVers
 
 // discoverReleases discovers new releases from the startChannels for the given arch.
 // It returns a ReleasesByChannel, with keys as full channel names.
-func discoverReleases(client *http.Client, graphURL *url.URL, startChannel string, arch string) (ReleasesByChannel, error) {
+func discoverReleases(client *http.Client, graphURL *url.URL, startChannel string, arch string, allowedConditionalEdgeRisks []string) (ReleasesByChannel, error) {
```

Comment:
this is probably a question for @wking - are the risk names unique across all channels?

Reply:
discoverReleases is per a channel "group", but the question remains valid - will drive the usage.